bridges/centralized-ethereum/src/actors/wit_poller.rs

```diff
@@ -79,9 +69,21 @@ impl WitPoller {
             let current_timestamp = get_timestamp();
 
             for (dr_id, dr_bytes, dr_tx_hash, dr_tx_creation_timestamp) in pending_drs {
-                let report = witnet_client.execute("dataRequestReport", json!([dr_tx_hash]));
+                let method = String::from("dataRequestReport");
+                let params = json!([dr_tx_hash]);
+                let req = jsonrpc::Request::method(method)
+                    .timeout(Duration::from_millis(5_000))
+                    .params(params)
+                    .expect("params failed serialization");
+                let report = witnet_client.send(req).await;
+                let report = match report {
```

Comment:
I think the report is matched twice, in line 79 and in line 87, is it intentional?

Reply:
It's because the report is a `Result<Result<_>>`, so this match unwraps the outer error.